### PR TITLE
Fix #4963, change log level from debug to info

### LINKED ISSUE #4963: Spark and BigDL Log level is set to DEBUG
Log level become DEBUG and cannot be changed by env and conf.
Is this due to https://github.com/intel-analytics/BigDL/blob/main/scala/dllib/src/main/resources/log4j2-test.xml#L22 ?

### PR DESCRIPTION
**Pull request by 00-Luoshu**
* Fix the error that log level cannot be changed by env and conf

### CODE REVIEW

File: scala/dllib/src/main/resources/log4j2-test.xml [hunk 1]
```diff
@@ -19,9 +19,11 @@
...
-        <Root level="debug">
+        <Root level="info">
             <AppenderRef ref="Console"/>
-            <AppenderRef ref="Routing" level="debug" />
         </Root>
+        <Logger name="org" level="info" additivity="false">
+            <AppenderRef ref="Console"/>
+        </Logger>
     </Loggers>
 </Configuration>
```
Comment (qiyuangong) [inline scala/dllib/src/main/resources/log4j2-test.xml:29]: Add an empty line at the end

File: scala/dllib/src/main/resources/log4j2-test.xml [hunk 2]
```diff
@@ -19,9 +19,11 @@
         </Routing>
     </Appenders>
     <Loggers>
-        <Root level="debug">
+        <Root level="info">
             <AppenderRef ref="Console"/>
-            <AppenderRef ref="Routing" level="debug" />
```
Comment (qiyuangong) [inline scala/dllib/src/main/resources/log4j2-test.xml:24]: Add it back, and change to info

**Review by qiyuangong — APPROVED** [review]
LGTM

**Comment by dding3** [comment]
Please run dllib jekins http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-PR-Dllib-Validation/ to ensure if doesn't fail anything before merge this PR.

**Comment by 00-Luoshu** [comment]
http://10.112.231.51:18889/view/BigDL-PR-Validation/job/BigDL-PR-dllib-Integration-ExampleTest/22/  build pass

**Comment by dding3** [comment]
DLLIB jekins also pass. http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-PR-Dllib-Validation/529/
http://10.112.231.51:18888/job/BigDL-PRVN-pythontest-Spark-3.1/568/
http://10.112.231.51:18888/job/BigDL-PRVN-pythontest-Spark-2.4-py36-part2/218/

I think we can merge this pr now.

**Pull request merged**